app/utils/kcidb.py

```diff
@@ -190,6 +190,7 @@ def push_tests(group_id, bq_options, db_options={}, db=None):
         url = "/".join([
             STORAGE_URL,
             build[models.FILE_SERVER_RESOURCE_KEY],
+            group[models.LAB_NAME_KEY],
```

Comment:
Right, this is the bit where we're duplicated some code from `app/utils/callback/lava.py` in the `_get_directory_path()` function:
```python
    if file_server_resource:
        directory_path = os.path.join(
            base_path,
            file_server_resource,
            meta[models.LAB_NAME_KEY])
```
I think one way to improve this would be to store a file_server_resource value which includes the lab name in the test group document, but that can be consolidated in a follow-up PR.

So this fix is probably the best one we can get at this point.  Merging now and starting another round of tests on staging.